tools/c7n_index/c7n_index/metrics.py

```diff
@@ -128,35 +145,47 @@ def get_indexer(config, **kwargs):
 @indexers.register('es')
 class ElasticSearchIndexer(Indexer):
 
-    def __init__(self, config, **kwargs):
+    def __init__(self, config):
         self.config = config
-        self.es_type = kwargs.get('type', 'policy-metric')
-
-        host = [config['indexer'].get('host', 'localhost')]
+        # self.es_type = kwargs.get('type', 'policy-metric')
```

Comment:
Commenting this line out will affect the code here:
https://github.com/salwandhruvdev/cloud-custodian/blob/e62d4f555f4234f596c8d92820c46b5cae9b45eb/tools/c7n_index/c7n_index/metrics.py#L181.

Reply:
Un-commented this line